storage/aptosdb/src/lib.rs

```diff
@@ -501,13 +510,13 @@ impl AptosDB {
                 "state_merkle_db_sec",
                 state_merkle_db_column_families(),
             )?,
-            if rocksdb_configs.use_kv_db {
+            if rocksdb_configs.use_state_kv_db {
                 Some(DB::open_cf_as_secondary(
-                    &gen_rocksdb_options(&rocksdb_configs.kv_db_config, false),
-                    kv_db_primary_path,
-                    kv_db_secondary_path,
-                    "kv_db_sec",
-                    kv_db_column_families(),
+                    &gen_rocksdb_options(&rocksdb_configs.state_kv_db_config, false),
```

Comment:
See https://github.com/facebook/rocksdb/wiki/Read-only-and-Secondary-instances

I don't think we currently use it because Alden observed some unexpected behaviors which we don't fully understand. But let's keep it in case we need it in the future.